lib/playground.dart

```diff
@@ -397,8 +402,23 @@ class Playground implements GistContainer {
     }
   }
 
+  void _checkForHtml() {
+    if (_context.htmlDocument.value.trim().isEmpty && !htmlIsEmpty) {
+      htmlIsEmpty = true;
+      querySelector("#consoletab").style.display = "none";
+      querySelector("#resulttab").style.display = "none";
+      _outputpanel.style.display = "block";
+      _frame.style.display = "none";
+    } else if (_context.htmlDocument.value.trim().isNotEmpty && htmlIsEmpty){
+      htmlIsEmpty = false;
+      querySelector("#consoletab").style.display = "block";
+      querySelector("#resulttab").style.display = "block";
+      _toggleResultTab();
+    }
+  }
+
   void _handleRun() {
-    _toggleConsoleTab();
+//    _toggleConsoleTab();
```

Comment:
Please remove code rather than commenting out - unless there is a reason for it to remain